docs/REST API/rest-api.md

```diff
@@ -5501,6 +5484,59 @@ solely on the response status code returned by this endpoint.
 
 `Status: 200`
 
+### Resend custom OS setting (configuration profile)
+
+Resends a configuration profile for the specified host.
+
+`POST /api/v1/fleet/hosts/:id/configuration_profiles/:profile_uuid/resend`
+
+#### Parameters
+
+| Name | Type | In | Description |
+| ---- | ---- | -- | ----------- |
+| id   | integer | path | **Required.** The host's ID. |
+| profile_uuid   | string | path | **Required.** The UUID of the configuration profile to resend to the host. |
+
+#### Example
+
+`POST /api/v1/fleet/hosts/233/configuration_profiles/fc14a20-84a2-42d8-9257-a425f62bb54d/resend`
+
+##### Default response
+
+`Status: 202`
+
+
+### Batch-resend custom OS setting (configuration profile)
+
+
+`POST /api/v1/fleet/configuration_profiles/resend/batch`
+
+#### Parameters
+
+| Name    | Type    | In   | Description                                                                                                                                                                                                                                                                                                                        |
+| ------- | ------- | ---- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| profile_uuid | integer | body | **Required**. The UUID of the existing configuration profile you'd like to resend.|
+| filters | object | body | **Required**. Currently, only the `profile_status=failed` filter is supported.|
```

Comment:
I know  it may feel like overkill, but I'd consider formatting this the same way as [this endpoint](https://fleetdm.com/docs/rest-api/rest-api#batch-delete-hosts-by-filter-or-ids) for consistency.

```suggestion
| filters | object  | body | **Required**. See [filters](#filters)  |

##### Filters

| Name                              | Type    | Description   |
| -----------------------| ------- | ----------------------------------------------------------------------------------- |
| profile_status                | string   | Profile status. Currently, `"failed"` is supported. |
```

Reply:
Thanks!